docs/user_manual.md

```diff
@@ -272,7 +273,11 @@ For example, `{"foo": {"internalPort": 4711}}`  will  map the internal port
 4711 of the container to an arbitrary external port on the host. 
   
 Specifying `{"foo": {"internalPort": 4711, "externalPort": 80}}` will map
-internal port 4711  of the container to port 80 on the host. 
+internal port 4711  of the container to port 80 on the host.
+
+The `ip` attribute is optional and specifies the IP address on which the port
+will be exposed. Although Helios will check the IP string is a valid IP address,
+it won't verify the address is routable or whether it'll work at deploy-time.
```

Comment:
I think it's perfectly fine as it is now. Personally, I find it clearer if you speak about interfaces but that's just me.

For inspiration, this is what the unbound docs say:

```
       interface: <ip address[@port]>
              Interface  to  use  to connect to the network. This interface is
              listened to for queries from clients, and answers to clients are
              given  from  it.  Can be given multiple times to work on several
              interfaces. If none are given the default is to listen to local-
              host.   The  interfaces  are not changed on a reload (kill -HUP)
              but only on restart.  A port number can be specified with  @port
              (without spaces between interface and port number), if not spec-
              ified the default port (from port) is used.

       ip-address: <ip address[@port]>
              Same as interface: (for easy of compatibility with nsd.conf).
```

and nginx (no mention of interfaces here!):

```
Syntax:	listen address[:port] ...
...
Sets the address and port for IP, or the path for a UNIX-domain socket on which the server will accept requests. Both address and port, or only address or only port can be specified. An address may also be a hostname, for example:
```